### PR TITLE
Misc improvements

### PR DESCRIPTION
**Pull request by ddabble**
### Highlights:
* Added a couple missing migrations from earlier PRs (99288bbe4437866a30d817dfb511be3403e7a3ef)
* Switched from using `os.path` to using `pathlib` (9137dd5c59da5236eeabf03876e6eef45f8fc3e3)
* Added website theme color (8a911825191318b641078c7e16e0df27c6f428a7)
* Added lots of format strings (2f4d0cb2602a070414630849b0e765bd646063a4)
* Replaced some magic numbers (42f322325d353f6d3804c5cd0fcf84e2f7aa8d25)
* Big code cleanup (5343793d0f92b9993eec0e2f5f68573a4eb8b741)

### CODE REVIEW

File: web/tests/test_permission_tags.py [hunk 1]
```diff
@@ -5,7 +5,6 @@
 
 
 class HasAnyPermissionTest(TestCase):
-
```
Comment (zootos) [inline web/tests/test_permission_tags.py]: This line should not be removed

File: web/tests/test_semantic_ui_calendar_extra.py [hunk 2]
```diff
@@ -1,31 +1,29 @@
...
 
-import pytz
 from django.test import TestCase
 from django.utils import timezone
 
 from web.templatetags.semantic_ui_calendar_extra import set_current_date
 
 
 class TemplateTagTestCases(TestCase):
-
```
Comment (zootos) [inline web/tests/test_semantic_ui_calendar_extra.py]: This line should not be removed

File: checkin/views.py [hunk 3]
```diff
@@ -264,7 +263,6 @@ def card_id_valid(self, card_id):
 
 
 class RegisterProfileView(TemplateView):
-
```
[the remark this answers is not in the document]
Comment (ddabble) [reply checkin/views.py]: Most other classes before this change had no empty line there; should I add an empty line to all other classes as well?

[the remark this answers is not in the document]
Comment (zootos) [reply checkin/views.py]: You could. Remember that there should only be an empty line if the first thing that appears in the class is a method.

[the remark this answers is not in the document]
Comment (ddabble) [reply checkin/views.py]: All right, thanks for the clarification! 🙂

**Pull request merged**